lib/app.js

```diff
@@ -30,26 +30,28 @@ app.use(function* () {
     this.throw("Expected a source file", 400);
   }
   const sourceFilePath = parts.file.source.path;
+  const sourceDirPath = path.dirname(parts.file.source.path);
 
-  if (!parts.file.caniuse) {
-    this.throw("Expected caniuse file", 400);
-  }
-  const caniuseFilePath = parts.file.caniuse.path;
-
-  if (!parts.file.w3cbugs) {
-    this.throw("Expected w3cbugs file", 400);
-  }
-  const w3cbugsFilePath = parts.file.w3cbugs.path;
+  const quiet = parts.field.quiet !== undefined ? "--quiet" : "";
```

Comment:
OK, will change it to that, then. I just wasn’t familiar enough with the simplest way to get the query params in a node app.

Incidentally, as long as we are putting the params in the query string, we could actually just send the document as the POST entity body, and drop the need to send it as multipart/form-data. But if we did that, I don’t know what equivalent module there might be for node that provides the convenience that this co-multipart module does, as far as saving the file and exposing a property to get to it directly.